_includes/app-self_signed_ca.md

```diff
@@ -1,32 +1,19 @@
 ## Using a Self-Signed CA Certificate
 
-Adding a self-signed Certificate Authority (CA) certificate for SSL
-authentication requires additional configuration on OpenStack Platform
-and Microsoft System Center Virtual Machine Manager (SCVMM) providers.
+Adding a self-signed Certificate Authority (CA) certificate for SSL authentication requires more configuration on OpenStack Platform and Microsoft System Center Virtual Machine Manager (SCVMM) providers.
 
-<div class="note">
-
-This procedure is not required for OpenShift Container Platform, Red Hat
-Virtualization, or middleware manager providers, which have the option
-to select **SSL trusting custom CA** as a **Security Protocol** in the
-user interface. These steps are needed only for providers without this
-option in the user interface.
-
-</div>
+**Note**: This procedure is not required for OpenShift Container Platform, Red Hat Virtualization, or middleware manager providers, which have the option to select **SSL trusting custom CA** as a **Security Protocol** in the user interface. These steps are needed only for providers without this option in the user interface.
```

Comment:
It can be done in a follow up PR, but middleware manager providers no longer exist.  We can remove all instances of that wording in the docs.